DispatcherService/compose/docker-compose-old.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  mongo:
+    image: mongo:3.2
+    volumes:
+      - mongo-data:/data/db
+    expose:
+      - "27017"
+  app:
+    image: mydocker2112/dispatcher-service-pm2
+    ports:
+      - "3000:3000"
+    links:
+      - mongo
+    depends_on:
+      - mongo
+    environment:
+      - MONGO_DB_CONN_STRING=mongo:27017/Uber
+      - LOGGLY_TOKEN=4aa61359-b9ea-47ce-a8e9-50da65ffd114
```

Comment:
It's a very bad practise to push secrets to git, potentially an attacker could use this and fuck you up with shit loads of bill from loggly.. use some kind of param store from GCP

Reply:
yep..agreed....created a task for this....will change this